fab/fabfile.py

```diff
@@ -462,8 +453,8 @@ def update_code(use_current_release=False):
 
     with cd(env.code_root if not use_current_release else env.code_current):
         sudo('git remote prune origin')
-        sudo('git fetch')
-        sudo("git submodule foreach 'git fetch'")
+        sudo('git fetch origin {}'.format(env.code_branch))
+        sudo("git submodule foreach 'git fetch origin {}'".format(env.code_branch))
```

Comment:
Just spoke with @benrudolph about this - I think you can just drop the `git submodule foreach` thing altogether and let `git submodule update` take care of it.